codegen/module.go

```diff
@@ -1787,6 +1789,8 @@ type ModuleInstance struct {
 	mu                sync.RWMutex
 	// QPSLevels is map of circuit breaker name to qps level for all circuit breakers
 	QPSLevels map[string]int
+	// ProxyTemplates is a collection of proxy templates apart from zanzibar
```

Comment:
if using custom templates. we can write collection of custom templates that can be passed to zanzibar

Reply:
Done